File: SummaMove/app/Models/Oefening.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Oefening extends Model
{
    protected $table = 'oefening';
    public $timestamps = false;

    protected $fillable = [
        'name',
        'description',
        'description_en',
        'image'
    ];
}

File: SummaMoveApp/lib/oefening.dart
// oefening.dart
class Oefening {
  final String name;
  final String description;
  final String image;

  Oefening({required this.name, required this.description, required this.image});

  factory Oefening.fromJson(Map<String, dynamic> json) {
    return Oefening(
      name: json['name'],
      description: json['description'],
      image: json['image'],
    );
  }
}